### ISSUE TITLE
[📍] Use an older revision for perf try jobs

### ISSUE THREAD

**Issue by dave-2** [catapult-project/catapult#4186]
Right now perf try jobs run on HEAD and HEAD+patch. Since builds take some time, HEAD usually doesn't have pre-builts already available, so every perf try job kicks off an extra build. That doubles the load on the builders. We'd like to pick older revisions for perf try jobs.

**Comment by simonhatch** [comment]
Yeah we could probably just pick last one run by the perf bots or something.
